src/icons/js-frontend-icon.tsx

```diff
@@ -1,32 +1,6 @@
-import { FC } from 'react';
-import { IconProps } from './icon.types';
+import jsFrontend from '@/assets/svg/JsFrontend.svg';
+import Image from '@/features/image';
 
-export const JsFrontendIcon: FC = ({ width = '48', height = '36' }: IconProps) => (
-  <svg
-    width={width}
-    height={height}
-    viewBox="0 0 40 47"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg">
-    <path
-      d="M2.64431 39.7828C2.75249 41.1049 3.67162 42.2203 4.94873 42.5792L19.1832 46.5797C19.753 46.7398 20.3561 46.7374 20.9246 46.5728L34.6982 42.5846C35.9583 42.2197 36.864 41.1172 36.9773 39.8103L40.094 3.86042C40.2545 2.00904 38.797 0.416485 36.9387 0.412663L2.86761 0.342598C1.0102 0.338778 -0.453471 1.92383 -0.301996 3.77506L2.64431 39.7828Z"
-      fill="#757575"
-      stroke="url(#paint0_linear_20960_6995)"
-      strokeWidth="1.26946"></path>
-    <path
-      d="M18.2511 8.66396C18.2511 8.39843 18.0359 8.18318 17.7703 8.18318H14.4212C14.1556 8.18318 13.9404 8.39843 13.9404 8.66396V32.5086C13.9404 32.8102 13.6659 33.0373 13.3696 32.9809L10.9784 32.5254C10.7517 32.4822 10.5876 32.2839 10.5876 32.0531V29.5081C10.5876 29.2719 10.416 29.0707 10.1828 29.0334L7.15293 28.5486C6.86081 28.5019 6.59619 28.7275 6.59619 29.0234V35.5917C6.59619 35.8106 6.74404 36.0019 6.95587 36.057L17.6492 38.8402C17.9538 38.9195 18.2511 38.6896 18.2511 38.3749V8.66396ZM21.4848 8.66396C21.4848 8.39843 21.7 8.18318 21.9656 8.18318H32.8506C33.1478 8.18318 33.3738 8.45013 33.3248 8.74325L32.6873 12.5554C32.6486 12.7871 32.448 12.9569 32.2131 12.9569H26.5212C26.2557 12.9569 26.0404 13.1722 26.0404 13.4377V20.3098C26.0404 20.5754 26.2557 20.7906 26.5212 20.7906H32.1145C32.3896 20.7906 32.6086 21.0209 32.5947 21.2956L31.8402 36.2398C31.8294 36.4542 31.6777 36.6354 31.4685 36.6839L22.0741 38.8603C21.7727 38.9302 21.4848 38.7013 21.4848 38.392V34.3909C21.4848 34.1855 21.6153 34.0028 21.8096 33.9361L28.165 31.757C28.349 31.6938 28.4769 31.526 28.4889 31.3317L28.8361 25.6969C28.8548 25.3925 28.5891 25.1478 28.2873 25.1915L22.0345 26.0969C21.7445 26.1388 21.4848 25.914 21.4848 25.6211V8.66396Z"
-      fill="white"></path>
-    <defs>
-      <linearGradient
-        id="paint0_linear_20960_6995"
-        x1="9.93003"
-        y1="48.3752"
-        x2="39.6493"
-        y2="0.971646"
-        gradientUnits="userSpaceOnUse">
-        <stop stopColor="white" stopOpacity="0.85"></stop>
-        <stop offset="1" stopColor="white" stopOpacity="0.35"></stop>
-      </linearGradient>
-    </defs>
-  </svg>
-);
+export const JsFrontendIcon = () => {
+  return <Image src={jsFrontend} alt="JsFrontend icon" />;
```

Comment:
Could you please clarify where the { width = '48', height = '36' } code snippet has been relocated? The dimensions of the image seem to be missing.

Reply:
This component has been removed because it is not used on the project.